Userland/Applications/Browser/CookiesModel.cpp

```diff
@@ -8,10 +8,10 @@
 
 namespace Browser {
 
-void CookiesModel::add_item(Web::Cookie::Cookie const& item)
+void CookiesModel::set_items(AK::Vector<Web::Cookie::Cookie> items)
 {
     begin_insert_rows({}, m_cookies.size(), m_cookies.size());
-    m_cookies.append(item);
+    m_cookies = items;
```

Comment:
No need to copy the vector here

```suggestion
    m_cookies = move(items);
```